3_javascript/javascript3/week1/WeatherApp/weatherapp.js

```diff
@@ -0,0 +1,59 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const cityNameElement = document.getElementById('city-name');
+    const temperatureElement = document.getElementById('temperature');
+    const weatherIconElement = document.getElementById('weather-icon');
+    const windSpeedElement = document.getElementById('wind-speed');
+    const sunriseElement = document.getElementById('sunrise');
+    const sunsetElement = document.getElementById('sunset');
+    const getWeatherBtn = document.getElementById('get-weather-btn');
+    const unitSelector = document.getElementById('unit-selector');
+
+    getWeatherBtn.addEventListener('click', function () {
+        const cityInput = prompt('Enter city name:');
+        if (cityInput) {
+            fetchWeatherData(cityInput);
+        }
+    });
+
+    function fetchWeatherData(city, unit = 'metric') {
+        fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=${unit}&appid=08fc1fab698b506e105eaf9a8a929a11`)
```

Comment:
I don't know if you learned already about storing the API key in an `.env` file because right now the API key is hardcoded in your script. This is a security risk as others can misuse it.

Reply:
Unfortunately, I don't know anything about the .env file yet.